mypy/sharedparse.py

```diff
@@ -0,0 +1,8 @@
+"""Shared logic between our three mypy parser files.
```

Comment:
Style nit: In single-line docstrings, we keep both `"""`s on the same line (e.g. `"""Doc string."""`).